docs/reference/developer/getting-started.rst

```diff
@@ -101,6 +101,25 @@ In Python:
 
   import scipp as sc
 
+As an alternative to ``conda develop``, some developers may prefer using ``pip`` to make an editable install.
```

Comment:
It is not an alternative. You can use `conda develop src` to make an editable install after installing with cmake.